app/javascript/packs/flagUserModal.jsx

```diff
@@ -0,0 +1,212 @@
+import { h, render } from 'preact';
+import { useState, useRef } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { request } from '../utilities/http';
+import { Button } from '@crayons/Button/Button';
+
+async function confirmFlagUser({ reactableType, category, reactableId }) {
+  const body = JSON.stringify({
+    reactable_type: reactableType,
+    category,
+    reactable_id: reactableId,
+  });
+
+  try {
+    const response = await request('/reactions', {
+      method: 'POST',
+      body,
+    });
+
+    const outcome = await response.json();
+
+    if (outcome.result === 'create') {
+      // eslint-disable-next-line no-restricted-globals
```

Comment:
I think these lines were meant to disable `alert()`, now that's not there we don't need them